ibm_catalog.json

```
@@ -134,6 +142,29 @@
                 }
               ]
             },
+            {
+              "key": "enable_platform_metrics",
+              "type": "boolean",
```

another workaround needed for this one:
```suggestion
              "type": "bool",
```